src/components/utils/test/convert_utils_test.cc

```diff
@@ -30,26 +31,53 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef SRC_COMPONENTS_INCLUDE_TEST_PROTOCOL_HANDLER_MOCK_TELEMETRY_OBSERVER_H_
-#define SRC_COMPONENTS_INCLUDE_TEST_PROTOCOL_HANDLER_MOCK_TELEMETRY_OBSERVER_H_
-
-#include "gmock/gmock.h"
-
-#include "protocol_handler/telemetry_observer.h"
+#include "gtest/gtest.h"
+#include "utils/convert_utils.h"
 
 namespace test {
 namespace components {
-namespace protocol_handler_test {
+namespace utils_test {
+
+using namespace ::utils;
+
+TEST(ConvertUtilsTest, ConvertInt64ToLongLongInt_CorrectValue) {
+  int64_t value_to_convert = 42;
+  EXPECT_EQ(typeid(long long int),
+            typeid(ConvertInt64ToLongLongInt(value_to_convert)));
+}
+
+TEST(ConvertUtilsTest, ConvertLongLongIntToInt64_CorrectValue) {
+  long long int value_to_convert = 42;
+  EXPECT_EQ(typeid(int64_t),
+            typeid(ConvertLongLongIntToInt64(value_to_convert)));
+}
+
+TEST(ConvertUtilsTest, ConvertUInt64ToLongLongUInt_CorrectValue) {
+  uint64_t value_to_convert = 42;
+  EXPECT_EQ(typeid(unsigned long long int),
+            typeid(ConvertUInt64ToLongLongUInt(value_to_convert)));
+}
+
+TEST(ConvertUtilsTest, ConvertLongLongUIntToUInt64_CorrectValue) {
+  unsigned long long int value_to_convert = 42;
+  EXPECT_EQ(typeid(uint64_t),
+            typeid(ConvertLongLongUIntToUInt64(value_to_convert)));
+}
+
+TEST(ConvertUtilsTest, ConvertBinaryDataToString_ValidCharacteres_CorrectText) {
+  const uint8_t data[] = {'s', 'u', 'c', 'c', 'e', 's', 's'};
+  const size_t data_size = 7;
+  EXPECT_EQ("success", ConvertBinaryDataToString(&data[0], data_size));
```

Comment:
@mkorniichuk I believe you should change the const literal `"success"` to a predefined const `std::string = "success"`, because you check the method that convert a data to the `std::string`.